glean-core/ios/Glean/Glean.swift

```diff
@@ -328,8 +328,8 @@ public class Glean {
 
     /// Handle background event and submit appropriate pings
     func handleBackgroundEvent() {
-        Pings.shared.baseline.submit()
-        Pings.shared.events.submit()
+        Pings.shared.baseline.submit(Pings.shared.baselineReasonCodes.background)
```

Comment:
Due to the way that the `submit` function is defined in the Swift bindings, you cannot omit the `reason:` label. You can also make this more Swifty by just using `.background` and letting Swift infer the rest.  Like this:
```Swift
Pings.shared.baseline.submit(reason: .background)
Pings.shared.events.submit(reason: .background)
```

Reply:
Also, thanks for catching the missing baseline reason code :)

Reply:
> Also, thanks for catching the missing baseline reason code :)

Well, in fairness, there's only one "reason" for the baseline ping in Swift since it's more predictable, but I think it's nice to be consistent here.

Reply:
I think there are still ways to get the `.dirty` reason for the baseline ping, if the app crashed, for instance.

Reply:
Oh, indeed.  I guess I didn't realize that...  Good to know.